### PR TITLE
Generated from SPDK v23.05-pre-784-g7a50a6bc4

### PR DESCRIPTION
**Pull request by askervin**

### CODE REVIEW

**Comment by askervin** [comment]
This is an example of a pull request when created as described in README.

We probably wish to wait for SPDK release before merging this. It'll give a cleaner commit message, yet there would not be any changes in the content.

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+go 1.19
+
+require (
+	google.golang.org/grpc v1.55.0
+	google.golang.org/protobuf v1.30.0
+)
+
+require (
+	github.com/golang/protobuf v1.5.3 // indirect
```
Comment (avalluri) [inline go.mod:11]: @askervin  Is't golang/protobuf is deprecate? Instead we should use  `google.golang.org/protobuf` I guess.

Comment (avalluri) [reply go.mod:11]: Okay got it. These go dependencies are coming from spdk. Please ignore my comment.

**Comment by askervin** [comment]
Closing. Merged via review.spdk.io.

**Pull request closed without merging**